### PR TITLE
vxlan: create vxlan setup example, clean up interfaces page

### PR DESCRIPTION
**Pull request by Monviech**
- Cleans up the interface page by grouping elements, removing bold titles etc...
- Creates a VXLAN setup tutorial

Fixes: https://github.com/opnsense/docs/issues/598

### CODE REVIEW

File: source/manual/how-tos/dt_ger_iptv.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-**Deutsche Telekom Germany IPTV (Magenta TV) setup**
-========================================================
+Deutsche Telekom Germany IPTV (Magenta TV) setup
+================================================
```
Comment (AdSchellevis) [inline source/manual/how-tos/dt_ger_iptv.rst:2]: @Monviech I usually make these a it lengthier to prevent having to change them constantly.

**Review by AdSchellevis — APPROVED** [review]
It might be good to refer to the guide in `other-interfaces.rst` too for clarity.

**Comment by Monviech** [comment]
@AdSchellevis Thanks. I will make these a bit longer, makes sense.

I'll also add a reference. :)

File: source/manual/how-tos/vxlan_bridge.rst [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+------------------
+
+- | Go to :menuselection:`Interfaces --> Other Types --> VXLAN` and create ``vxlan1`` interfaces:
+       
+===============  ================  ================
+**Option**       **Site A**        **Site B**
+===============  ================  ================
+VNI              1                 1
+Source address   172.16.88.1/32    172.16.88.2/32
+Remote address   172.16.88.2/32.   172.16.88.1/32
```
Comment (Monviech) [inline source/manual/how-tos/vxlan_bridge.rst:105]: stray dot

**Pull request merged**